### ISSUE TITLE
Problems with BLK format

### ISSUE THREAD

**Issue by Livan89** [openMVG/openMVG#1703]
Good afternoon.

I am trying to generate a model from BLK format images. The images are a sequence of a building. The following happens to me:

The corners of the building appear only in two images. The images are textured and extract many key points, however the corners do not appear in the sparse. any ideas?

Thank you.

**Comment by pmoulon** [comment]
Hello Livan,
1. Can you describe with BLK you are using (BLK 360 images or rectilinear)
2. SfM will triangulate the points that are matches between multiple images, sometimes points are detected from 2 images, sometimes more. It could be possible that your corner is not matched or does not triangulate with enough accuracy to be kept in the 3d point cloud.

Here what I would advise:
- Try to extract more keypoint `-p HIGH` in ComputeFeatures
- You can also try to densify the point cloud using MVS (PMVS, OpenMVS) and see more geometry in the output
- If you want to triangulate point that you click manually you can do it using the ControlPoint UI

Feel free to share your images to openmvg-team[AT]googlegroups.com if you want more feedback from our team on your specific dataset

Note, if 360:
- Did you checked that your BLK image are perfectly equirectangular (360x180), if not you can add black band on top and bottom to fill back the Field Of View

**Comment by Livan89** [comment]
Hello Pierre Moulon. Thank you very much for your answer.

I had already worked on the ideas you give me.

I am using rectilinear. I have used key point extraction with HIGH, ULTRA and even worked on some internal modifications to extract more points, however the corners are still not shown. I used openMVG_main_exportMatches.exe to display the matches between points, both images in the corner have a good match between them, but the right extreme (i.e. the corner that is not reconstructed in 3D) only has matches between those two images,while the rest of these two images (the remaining area of ​​the image) has matches between them and with other images as well. I assume that the sfm assumes as problematic tracks or very short tracks those that unite the key points of only those two photos, but visually they do not seem bad, so it surprises me that they are not shown.

Sorry for my little knowledge of google groups, but I have no idea how to share the images in the group that you comment on, which I would like to do. I do not put them directly here because they are not mine, a friend is testing their BLK device and I am not authorized to make the images public, my apologies for that. If I could send them directly to a group or share them in some way more privative, I would be grateful to you.

Thank you.
